css/style.css

```diff
@@ -58,7 +58,11 @@ section {
     margin: 0 30px initial 30px;
 }
 
-footer {
+
+
+.footer {
```

Comment:
Like I mentioned, this should stay a tag and not be a class. I think bootstrap keeps the width at 100%, and the height should preferably not be a percentage.

Because it doesn't use the Bootstrap's way of making footers, it does weird things on high resolutions: https://puu.sh/vSuV0/81d8dbc641.png